docs/github.md

```diff
@@ -142,23 +149,24 @@ something like this:
 
 ```shell
 $ # Create a new commit on top of the second-to-last commit in `your-feature`,
-$ # as reviews requested a fix there.
+$ # as reviewers requested a fix there.
 $ jj new your-feature- # NOTE: the trailing hyphen is not a typo!
-$ # Address the comments by updating the code
-$ # Review the changes
+$ # Address the comments, by updating the code, then review the changes
```

Comment:
Should be consistent with whatever you do above. If you prefer this on a single line, that's fine, but I'm pretty sure the comma before the "by" should't be there. Also, even if these are on a single line, two sentences may be better.